apps/vr-tests/.storybook/addons.js

```diff
@@ -2,4 +2,5 @@
 import '@storybook/addons';
 import '@storybook/addon-options/register';
 import 'storybook-readme/register';
-import '@storybook/addon-actions/register';
\ No newline at end of file
+import '@storybook/addon-actions/register';
+import 'storybook-addon-rtl/register';
```

Comment:
hmmm. is this a new plugin?

Reply:
I might end up removing this. it wasn't working that well anyway.